generated/nifake/tests/test_session.py

```diff
@@ -763,14 +763,31 @@ def test_set_attribute_error(self):
     def test_add_properties_to_session_error(self):
         with nifake.Session('dev1') as session:
             try:
-                session.nonexistent_property = 5
+                session.non_existent_property = 5
                 assert False
-            except TypeError as e:
+            except AttributeError as e:
+                assert str(e) == "'Session' object has no attribute 'non_existent_property'"
+                pass
+            try:
+                value = session.non_existent_property  # noqa: F841
```

Comment:
Best to have more granular tests rather than a single test that tries several things.

Reply:
fixed